validator/specs/lib/validateTilesetSpec.js

```diff
@@ -42,7 +42,9 @@ var sampleTileset = {
 
 describe('validateTileset', function() {
     it('succeeds for valid tileset', function(done) {
-        expect(validateTileset(sampleTileset)
+        var tileset = clone(sampleTileset, true);
+        tileset.asset.version = '1.0';
+        expect(validateTileset(tileset)
```

Comment:
Instead of fixing the version here can you set to 1.0 above? This was my mistake.